### ISSUE TITLE
But where is the codee!?

### ISSUE THREAD

**Issue by righteousrenegade** [AllianceWars/alliancewars#7]
and is this from alliancewars.com? The greatest game to ever exist? Also, are you one of the original devs?

**Comment by kevinpwatson** [comment]
Yes, I'm a dev from alliancewars.com.  Just testing out GitHub.  The code may make it here one day.

**Comment by righteousrenegade** [comment]
Ahh, great. I used to play back in the day. Now I've been playing Eve online since AW doesn't really seem to be active anymore :(.  I really like github -- we use it a lot for my work. 

Where is the AW code currently hosted? And what language? I primarily work in Django but wouldn't mind learning some other frameworks and perhaps contributing.
